### PR TITLE
Fix integration test

### PR DESCRIPTION
**Pull request by kth13**
## What does this pull request do?

-Fixing some edge case failure in the integration testing

## What is the intent behind these changes?

- Some test cases might fail in the edge case scenario and this test is for fixing it and making it more robuse

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -127,6 +127,7 @@
     "jquery": "^3.6.1",
     "jwt-decode": "^3.1.2",
     "md5": "^2.3.0",
+    "moment": "^2.29.4",
```
Comment (sldblog) [inline package.json:130]: Can this live under `devDependencies`? Do we use it in production code?

**Review by sldblog — COMMENTED** [review]
Looks good, just one question about dev/prod dependencies

**Comment by sldblog** [comment]
`package-lock` changes seem to be missing, could those be added, please?

File: package.json (hunk 1 above)
Comment (kth13) [reply package.json]: Yes not needed. I have moved into ```devDependencies```

**Comment by sldblog** [comment]
Yes please. It contains a snapshot of fixed versions, so everyone can use the same exact versions.

The only thing that we have to watch out for is lockfile version: we should use npm that generates the same lockfile

**Comment by kth13** [comment]
> Yes please. It contains a snapshot of fixed versions, so everyone can use the same exact versions.
> 
> The only thing that we have to watch out for is lockfile version: we should use npm that generates the same lockfile

Thanks @sldblog , I have committed the other change you have suggested.

**Review by sldblog — APPROVED** [review]
